tests/files/test-vip-filesystem.php

```diff
@@ -386,4 +386,23 @@ public function data_get_transport_for_path(): iterable {
 			[ constant( 'WP_CONTENT_DIR' ) . '/languages/test.txt', 'direct' ],
 		];
 	}
+
+	public function test_wp_font_dir() {
+		// Only available in WP 6.5 and newer:
+		if ( ! function_exists( '\wp_get_font_dir' ) ) {
+			$this->markTestSkipped( 'test_wp_font_dir does not need to run for WP < 6.5.' );
+			return;
+		}
+
+		$font_dir = \wp_get_font_dir();
+
+		$this->assertEquals( $font_dir, [
+			'path'    => 'vip://wp-contentt/uploads/fonts',
```

Comment:
@sjinks do you happen to know where this double `t` in `wp-content` coming from?

Reply:
I was able to verify it happens here:

https://github.com/Automattic/vip-go-mu-plugins/blob/9cbad3ef1c15019c1ed5c0edf8a31dac6f88f1ae/files/class-vip-filesystem.php#L159-L167

If both `strlen( constant( 'WP_CONTENT_DIR' ) )` are changed to `strlen( constant( 'WP_CONTENT_DIR' ) ) + 1`, then it goes back to single t

Reply:
The root cause here is that WP_CONTENT_DIR constant can be set to a couple different things. So in `$params['path']`, it's `/tmp/...`, but in the constant itself, it's `/wp/...`. Since that's one character shorter, we see the off-by-one issue in the final string.

Not sure exactly why two different constants come into play, but this test file does set the `/wp` version, whereas most of the other tests have the `/tmp` version.

Reply:
for posterity, I fixed this by changing the WP_CONTENT_DIR  constant used by the vip-filesystem test to the `/tmp` version, which matches the constant elsewhere in the test suite